THIRD-PARTY-NOTICES.TXT

```diff
@@ -1110,6 +1110,13 @@ by Brian W. Kernighan and Rob Pike
 
 You may use this code for any purpose, as long as you leave the copyright notice and book citation attached.
 
+Note for code from Euclidean Affine Functions and Applications to Calendar
```

Comment:
```suggestion
Notice for Euclidean Affine Functions and Applications to Calendar
```

Reply:
This matches all the other entries. I see no reason for this entry to be different.